plugins/inputs/synproxy/synproxy.go

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2019 WorldStream B.V. All rights reserved.
```

Comment:
Can you remove this copyright notice, all code in this repo needs to be covered under the top level LICENSE.md.

Reply:
Ok clear, the noticed is removed.